.github/workflows/coding-standards.yml

```diff
@@ -0,0 +1,101 @@
+name: PHP Coding Standards
+
+on:
+    push:
+        branches:
+            - trunk
+            - 'release/**'
+            - 'wp/**'
+    pull_request:
+        paths:
+            # Any change to a PHP file should run checks.
+            - '**.php'
+            # These files configure Composer. Changes could affect the outcome.
+            - 'composer.*'
+            # This file configures PHPCS. Changes could affect the outcome.
+            - 'phpcs.xml.dist'
+            # Changes to workflow files should always verify all workflows are successful.
+            - '.github/workflows/*.yml'
+
+# Cancels all previous workflow runs for pull requests that have not completed.
+concurrency:
+    # The concurrency group contains the workflow name and the branch name for pull requests
+    # or the commit hash for any other events.
+    group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
+    cancel-in-progress: true
+
+jobs:
+    # Runs PHP coding standards checks.
+    #
+    # Violations are reported inline with annotations.
+    #
+    # Performs the following steps:
+    # - Checks out the repository.
+    # - Sets up PHP.
+    # - Configures caching for PHPCS scans.
+    # - Installs Composer dependencies.
+    # - Make Composer packages available globally.
+    # - Runs PHPCS on the full codebase with warnings suppressed.
+    # - Generate a report for displaying issues as pull request annotations.
+    # - Runs PHPCS on the `tests` directory without warnings suppressed.
+    # - Generate a report for displaying `test` directory issues as pull request annotations.
+    # - Ensures version-controlled files are not modified or deleted.
```

Comment:
Might be a convention in this repo, but to me this is just noise. Every step already has a name, so this is a maintenance nightmare in the making as people will forget to update this when they update the workflow steps.

Reply:
This description has been copied from https://github.com/WordPress/wordpress-develop/blob/trunk/.github/workflows/phpunit-tests.yml#L35
Personally, I don't think it's a noise, but I don't mind to remove it.
Removed in 2363c402862adec91e8429a53dd18756e228b990